api/src/test/java/org/apache/iceberg/transforms/TestNotStartsWith.java

```diff
@@ -106,23 +75,23 @@ public void testTruncateStringWhenProjectedPredicateTermIsLongerThanWidth() {
         .isEqualTo("ab");
 
     assertThat(evaluator.eval(TestHelpers.Row.of("abcde")))
-        .as("notStartsWith(abcde, truncate(abcde,2)) => false")
+        .as("notStartsWith('abcde', truncate(abcde,2)) => false")
         .isFalse();
 
     assertThat(evaluator.eval(TestHelpers.Row.of("ab")))
-        .as("notStartsWith(abcde, truncate(ab, 2)) => false")
+        .as("notStartsWith('ab', truncate('abcde', 2)) => false")
```

Comment:
I think the description before is unclear. I have modified it to a string value description.